File: Autolib/ClientApp/src/app/models/utilise.ts
import { Borne } from "./borne";
import { Client } from "./client";
import { Vehicule } from "./vehicule";

export class Utilise {
  idService: number;
  vehicule: Vehicule;
  client: Client;
  date: Date;
  borne_depart: Borne;
  borne_arrive: Borne;
}

File: Autolib/Modeles/Utilise.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Autolib
{
    public partial class Utilise
    {
        public int Vehicule { get; set; }
        public int Client { get; set; }
        public DateTime Date { get; set; }
        public int BorneDepart { get; set; }
        public int? BorneArrivee { get; set; }

        public virtual Borne BorneArriveeNavigation { get; set; }
        public virtual Borne BorneDepartNavigation { get; set; }
        public virtual Client ClientNavigation { get; set; }
        public virtual Vehicule VehiculeNavigation { get; set; }
    }
}
